tests/registry/jsr/@deno/otel/0.0.2/src/index.ts

```diff
@@ -1,38 +1,15 @@
 // Copyright 2024-2024 the Deno authors. All rights reserved. MIT license.
 
-import { context } from "npm:@opentelemetry/api@1";
-import {
-  BasicTracerProvider,
-  SimpleSpanProcessor,
-} from "npm:@opentelemetry/sdk-trace-base@1";
+import { context, trace, metrics } from "npm:@opentelemetry/api@1";
 
 // @ts-ignore Deno.telemetry is not typed yet
 const telemetry = Deno.telemetry ?? Deno.tracing;
```

Comment:
Is this on purpose? Are we gonna have both `Deno.telemetry` and `Deno.tracing`?

Reply:
No this was just to support both old and newer Deno versions - it is now superfluous and my follow up removes this entirely.